build-binutils.sh

```diff
@@ -0,0 +1,21 @@
+#/bin/bash
+
+VERSION=2.38
+
+# Build sed
+curl -LO --insecure https://ftp.gnu.org/gnu/binutils/binutils-$VERSION.tar.xz
```

Comment:
The `--insecure` is worrying here. Why is it needed? Can we verify a checksum too? Also, can this use long args for each?

Reply:
The reason we need it is because the CA certificates are too old in Debian 7.  Would we be satisfied with verifying checksums?  That should mitigate MITM attacks in the event of a compromised URL and would be a lot simpler than trying to update the CA certificates.